package.json

```
@@ -10,7 +10,7 @@
   "homepage": "https://github.com/comento/comento-ui",
   "scripts": {
     "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build --target lib --name comento-ui src/main.js",
+    "build": "vue-cli-service build --target lib --name comento-ui src/main.js && cp -R assets/style/base/color.scss dist && cp -R src/Elements/Core/Colors/index.js dist/color.js",
```

```suggestion
    "bundle": "vue-cli-service build --target lib --name comento-ui src/main.js",
    "copy": "cp -R assets/style/base/color.scss dist && cp -R src/Elements/Core/Colors/index.js dist/color.js",
    "build": "yarn bundle && yarn copy",
```
명령어를 나누면 어떨까요? 나중에 build가 왕창 길어질까봐 우려되서요.